cores/esp8266/sigma_delta.h

```diff
@@ -18,15 +18,112 @@
  License along with this library; if not, write to the Free Software
  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
  */
+ 
+ /*
+/******************************************************************************
+ * Info Sigma delta module
+
+This module controls the esp8266 internal sigma delta source
+Each pin can be connected to the sigma delta source
+The target duty and frequency can be modified via the register GPIO_SIGMA_DELTA
+
+THE TARGET FREQUENCY IS DEFINED AS:
+
+FREQ = 80,000,000/prescaler * target /256  HZ,     0<target<128
+FREQ = 80,000,000/prescaler * (256-target) /256  HZ,     128<target<256
+target: duty ,0-255
+prescaler: clk_div,0-255
+so the target and prescale will both affect the freq.
+
+Usage :
+1. sigma_delta_enable() : activate the sigma delta source with default prescalar (0) & target (0)
+2. sigma_delta_attachPin(pin), any pin 0..15, TBC if gpio16 supports sigma-delta source
+     This will set the pin to NORMAL output mode (pinMode(pin,OUTPUT))
+3. sigma_delta_setPrescaler(uint8_t) : reduce the output frequencies
+4. sigma_delta_setTarget(uint8_t) : set the output signal duty cycle, duty cycle = target/256
+
+5. sigma_delta_detachPin(pin), this will revert the pin to NORMAL output mode & GPIO source. 
+The sigma delta source remains on until :
+6. sigma_delta_disable()
+
+copy this code as example :
```

Comment:
This example should be made into a working example that builds with the rest of the core, so that at least build issues can be tracked.

Reply:
Good idea. The example would then be under libraries\esp8266\examples ?